footsoldiers/src/footsoldiers.lisp

```diff
@@ -910,6 +921,10 @@
   (write-char char (stream-of stream))
   (write-char char *standard-output*))
 
+(defmethod sb-gray:stream-write-sequence ((stream with-stdout) seq &optional (start 0) end)
+  (write-sequence seq stream start end)
+  (write-sequence seq *standard-output* :start start :end end))
```

Comment:
I'm not familiar with gray streams.  Why do we write to two streams here?

Reply:
I write to the file in addition to the output stream